### PR TITLE
Rough outline of international landing page.

### PR DESCRIPTION
**Pull request by lseelenbinder**
This is a very rough sketch of what we want.

It needs the following decisions and tasks to be complete:

- [ ] Determine if we want to show only countries that have an instance already or all countries.
- [x] Add a generic logo for the EU variant.
- [ ] Determine the proper footer content.
- [ ] Finish copy: https://hackmd.io/pHUNz_vnRtyiOSG2Sb_eAw?edit

### CODE REVIEW

**Comment by adriaandotcom** [comment]
I would do something like this:

<kbd>Screenshot</kbd>

Let's create a text here: https://hackmd.io/pHUNz_vnRtyiOSG2Sb_eAw?edit

File: app/locales/en.json [hunk 1]
```diff
@@ -1,12 +1,139 @@
...
-	"We can't find the page or service you're looking for": "We can't find the page or service you're looking for",
-	"The link you clicked may be broken, or the page no longer exists": "The link you clicked may be broken, or the page no longer exists",
-	"Something went wrong!": "Something went wrong!",
-	"Something must have gone wrong": "Something must have gone wrong",
-	"Send an e-mail to": "Send an e-mail to",
-	" if you can't get it to work.": " if you can't get it to work.",
-	"Share on": "Share on"
-}
\ No newline at end of file
+  "Postal code": "Postal code",
```
Comment (adriaandotcom) [inline app/locales/en.json]: Maybe checkout this whole file?

Comment (lseelenbinder) [reply app/locales/en.json]: I think I fixed that—I had to refresh the PR for it to show up, though.

Comment (adriaandotcom) [reply app/locales/en.json]: Cool!

**Comment by mihai-sysbio** [comment]
May I suggest to automatically present the user with a default website choice based on the [browser language](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/languages), if it exists in the locales list?

**Comment by amritnagi** [comment]
This is my high level thoughts for landing page

- A single ~Europe~ "worldwide" landing page with each country highlighted with a dot, which clicking it would take you to their site.
- Hovering over the dot will display the country name/flag

Ideally it would be great if we had a zoomable/scrollable map, if not - could have regional view's of the page which displays a bigger map (Europe, Asia etc).  Could have arrows to take the user to the next region

image

Check it out in Figma
https://www.figma.com/file/WriP7XWOZbni3X94J0asVX/CoronaStatus?node-id=6241%3A543

Love to hear people's thoughts (and volunteers to build it! :))

**Comment by lseelenbinder** [comment]
That would be fantastic! This has definitely outgrown what I was working originally.

I can do the map part, but I may not have extra cycles to build it until tomorrow. It's certainly doable, though.

**Pull request closed without merging**